packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -33,8 +33,12 @@ export class CDKDeployer implements BackendDeployer {
    */
   constructor(
     private readonly cdkErrorMapper: CdkErrorMapper,
-    private readonly backendLocator: BackendLocator
-  ) {}
+    private readonly backendLocator: BackendLocator,
+    private readonly packageManager: string
+  ) {
+    this.packageManager =
+      this.packageManager === 'npm' ? 'npx' : this.packageManager;
+  }
```

Comment:
A note for future follow up when we plumb this from CLI.

The deployer only cares about "package manager executable". So perhaps this resolution should happen in CLI.

Or, perhaps we should start thinking about moving package manager abstraction (controller?) to `platform-core` and have `PackageManagerController` injected here from CLI?

Reply:
Yea, we should move the abstruction to `platform-core`. I'll do it in a follow-up PR.

Reply:
added a comment d33f115b9e